extension/lib/emailjs/emailjs-mime-parser.js

```diff
@@ -810,10 +810,13 @@
             }
 
             // override charset for text nodes
-            this.charset = this.contentType.params.charset = 'utf-8';
+            // issue #3520 - do not override 7bit econdings and ISO-2022-JPxxx
+            if (!((this.headers['Content-Transfer-Encoding'] && this.headers['Content-Transfer-Encoding'].toLowerCase() === '7bit')
+                || (this.charset && this.charset.toUpperCase().startsWith('ISO-2022-JP')))) {
+                this.charset = this.contentType.params.charset = 'utf-8';
+            }
```

Comment:
have you checked if this method was run and the property `contentTransferEncoding` was set?
![image](https://user-images.githubusercontent.com/23335944/163834427-d3e45a55-7938-4afd-94f1-040c3e76842f.png)
Also, headers are probably keyed lower-cased